extensions/search-npm/package.json

```diff
@@ -132,7 +133,9 @@
   "dependencies": {
     "@raycast/api": "^1.83.2",
     "@raycast/utils": "^1.17.0",
+    "@types/git-url-parse": "^9.0.3",
```

Comment:
Let's move the DefinitelyTyped dependency to `devDependencies`.

Reply:
Updated, thank you.